packages/office-ui-fabric-react/src/components/ExtendedPicker/BaseExtendedPicker.tsx

```diff
@@ -25,8 +24,18 @@ export class BaseExtendedPicker<T, P extends IBaseExtendedPickerProps<T>>
   protected root = React.createRef<HTMLDivElement>();
   protected input = React.createRef<Autofill>();
   protected selection: Selection;
-  protected floatingPickerProps: IBaseFloatingPickerProps<T>;
-  protected selectedItemsListProps: IBaseSelectedItemsListProps<T>;
+
+  public static getDerivedStateFromProps(newProps: IBaseExtendedPickerProps<any>) {
```

Comment:
I think this should be removed now that you've removed `selectedItems` from state?

Reply:
Good call! I had mean to remove that and missed it.